### PR TITLE
SPARK-842. Maven assembly is including examples libs and dependencies

### PR DESCRIPTION
**Pull request by ghost**
Also, this commit includes (and depends upon) mllib module as represented in
  https://github.com/mesos/spark/pull/753

### CODE REVIEW

File: assembly/pom.xml [hunk 1]
```diff
@@ -62,43 +62,43 @@
...
       <groupId>org.spark-project</groupId>
-      <artifactId>spark-examples</artifactId>
+      <artifactId>spark-mllib</artifactId>
       <classifier>${classifier.name}</classifier>
-      <version>0.8.0-SNAPSHOT</version>
+      <version>${project.version}</version>
     </dependency>
     <dependency>
       <groupId>org.spark-project</groupId>
       <artifactId>spark-examples</artifactId>
```
Comment (mateiz) [inline assembly/pom.xml]: Looks like this and the one below are replicas of the spark-examples dependency for some reason. You should remove them.

Comment (ghost) [inline assembly/pom.xml]: One of them is for javadoc
Another is for sources

they are not dups.

Comment (mateiz) [inline assembly/pom.xml]: Oh, I see. Either way though we shouldn't include or depend on the spark-examples javadoc and sources.

Comment (mateiz) [inline assembly/pom.xml]: Maybe you meant to change those to mllib?

**Comment by ghost** [comment]
Removed examples completely.

File: assembly/pom.xml (hunk 1 above)
Comment (ghost) [inline assembly/pom.xml]: mllib is done separately. I have removed examples

**Comment by mateiz** [comment]
Okay, thanks!

**Pull request merged**